spinta/backends/__init__.py

```diff
@@ -24,7 +25,7 @@ def prepare(context: Context, type: Type, backend: Backend, value: object) -> ob
 
 
 @error.register()
-def error(exc: Exception, context: Context, backend: Backend, type: Type, **kwargs):
+def error(exc: Exception, context: Context, backend: Backend, type: MetaType, **kwargs):
```

Comment:
I think, here correct signature should be `type: Type`, because there is no such thing as `(Context, Backend, spinta.types.Type)`.

Reply:
Though `spinta.types.Type` itself shouldn't be removed?

Reply:
Yes, it can be removed, if no one else is using it.

Reply:
It's used here: https://github.com/atviriduomenys/spinta/blob/master/spinta/utils/response.py#L388

Reply:
There is should also be `spinta.types.type.Type`.